tests/app/test_jinja_filters.py

```diff
@@ -3,37 +3,39 @@
 
 from mock import Mock
 
-from app.jinja_filters import format_date, format_conditional_date, format_currency, format_multilined_string, \
-     format_percentage, format_start_end_date, format_household_member_name, format_str_as_date, \
-     format_str_as_date_range, format_str_as_month_year_date, format_number_to_alphabetic_letter, format_unit
+from app.jinja_filters import format_date, format_conditional_date, format_currency, get_currency_symbol, \
+     format_multilined_string, format_percentage, format_start_end_date, format_household_member_name, \
+     format_str_as_date, format_str_as_date_range, format_str_as_month_year_date, format_number_to_alphabetic_letter, \
+     format_unit
 
 
 class TestJinjaFilters(TestCase):  # pylint: disable=too-many-public-methods
 
-    def test_format_currency_without_decimals(self):
+    def test_get_currency_symbol(self):
         # Given
-        currency = 1
+        currency = "GBP"
 
         # When
-        format_value = format_currency(currency)
+        format_value = get_currency_symbol(currency)
 
-        self.assertEqual(format_value, '£1.00')
+        self.assertEqual(format_value, '£')
 
     def test_format_currency(self):
         # Given
-        currency = 1.12
+        currency = "GBP"
+        value = '11'
```

Comment:
The format_currency will accept any number of decimal places and will return it rounded to 2.
It would therefore be worth testing values with 2 and more decimal places.